index.js

```diff
@@ -1,45 +1,68 @@
-function Book(title, author) { this.title = title; this.author = author; }
-class Library {
-  constructor() {
-    if (window.localStorage.getItem('stored') === null) {
-      window.localStorage.setItem('stored', JSON.stringify([]));
-    }
-    this.books = JSON.parse(window.localStorage.getItem('stored'));
-    this.booklist = document.getElementById('booklist');
-    document.getElementById('add').addEventListener('click', () => { this.add(); });
-    this.show();
+/* eslint-disable max-classes-per-file */
+class Book {
```

Comment:
- [ ] The full implementation of the app still requires you to add local storage to preserve the items. The project should work the same way before you added the classes. Let's fix this